File: DisplaBackend/DisplaBackend/Models/ComprobanteItemRecargo.cs
using System;
using System.Collections.Generic;

namespace DisplaBackend.Models
{
    public partial class ComprobanteItemRecargo
    {
        public int Id { get; set; }
        public int IdRecargo { get; set; }
        public int IdComprobanteItem { get; set; }
        public decimal Monto { get; set; }

        public virtual ComprobanteItem IdComprobanteItemNavigation { get; set; }
        public virtual RecargoLente IdRecargoNavigation { get; set; }
    }
}

File: DisplaFrontend/src/app/model/comprobanteItemRecargo.ts
import { ComprobanteItem } from './comprobanteItem';
import { RecargoLente } from './recargoLente';

export interface ComprobanteItemRecargo {
    Id: number,
    IdRecargo: number,
    IdRecargoNavigation: RecargoLente,
    IdComprobanteItem: number,
    IdComprobanteItemNavigation: ComprobanteItem,
    Monto: number,
}
